src/types.db

```diff
@@ -33,6 +33,7 @@ compression             uncompressed:DERIVE:0:U, compressed:DERIVE:0:U
 compression_ratio       value:GAUGE:0:2
 commands                value:DERIVE:0:U
 connections             value:DERIVE:0:U
+connectivity            value:GAUGE:0:2
```

Comment:
This type is not needed to be added for this plugin. It sends only notifications, not metrics.

Reply:
This has been removed.